kubelet/datadog_checks/kubelet/kubelet.py

```diff
@@ -455,8 +457,10 @@ def _report_container_state_metrics(self, pod_list, instance_tags):
                 if self.pod_list_utils.is_excluded(cid, pod_uid):
                     continue
 
-                tags = instance_tags[:]
-                tags += tagger.tag('%s' % replace_container_rt_prefix(cid), tagger.ORCHESTRATOR) or []
+                tags = tagger.tag('%s' % replace_container_rt_prefix(cid), tagger.ORCHESTRATOR) or None
```

Comment:
```suggestion
                tags = tagger.tag(replace_container_rt_prefix(cid), tagger.ORCHESTRATOR)
```


- `or None` seems not needed
- `'%s' %` seems not needed if `cid` is a string